Core/src/music/v2mplayer.h

```diff
@@ -127,6 +131,7 @@ class V2MPlayer {
     // ------------------------------------------------------------------------------------------------------
 
 private:
+    Music& music;
```

Comment:
Referenser bör inte användas som class member. Vad händer när music går out of scope?